### DISCUSSION TITLE
Beta release in stable channel?

### DISCUSSION THREAD

**Discussion by fvdm** [AdguardTeam/AdGuardHome#7497] [General]
The [release notes](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.55) for v0.107.55 say it is a beta. Is that merely a copy/paste mistake or did a potentially unstable beta version end up in the stable channel?

> Note
> **This beta** includes a fix for...

**Comment by ainar-g** [comment]
It was a mistake on our part, should now be fixed.  Thanks!
